### PR TITLE
Add dependences installation instructions

### PR DESCRIPTION
**Pull request by azizmashkour**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,13 @@ AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
 Once you have set up your `.env` file, you need to run `./scripts/setup.sh` from the root folder in
 order to setup your credentials. This script will write the appropriate credentials to a
 `~/.aws/credentials` file that will allow you to deploy your application to the ecosystem.
+
+
+## Installing dependences
```
Comment (juancarlosfarah) [inline README.md]: Typo: **Dependencies**

File: README.md [hunk 2]
```diff
@@ -36,3 +36,13 @@ AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
...
+
+Make sure you have `Node`, `npm` and `yarn` installed on your local machine otherwise go to
```
Comment (juancarlosfarah) [inline README.md]: `node` should be in lowercase.

File: README.md [hunk 3]
```diff
@@ -36,3 +36,13 @@ AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
...
+`https://changelog.com/posts/install-node-js-with-homebrew-on-os-x` and install them; then run `yarn` from the project directory to install all dependences.
```
Comment (juancarlosfarah) [inline README.md]: You could mark this up as a link using markdown: `[name](url)`

File: README.md [hunk 4]
```diff
@@ -36,3 +36,13 @@ AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
...
+
+## Starting the server
+
+Navigate to the cloned or forked project directory in command line and type `npm start` and the project will automatically run on `localhost:3000`.
```
Comment (juancarlosfarah) [inline README.md]: Let's use `yarn` instead of `npm`.

Comment (juancarlosfarah) [inline README.md]: Minor change: "in command line" -> "using the command line, type... and the project..."

**Comment by azizmashkour** [comment]
Thanks @juancarlosfarah for your comments. I just fixed the typos.

**Pull request merged**